.travis.yml

```diff
@@ -2,6 +2,7 @@ language: python
 python:
   - "3.6"
   - "3.7"
+  - "pypy3.6"
```

Comment:
Add `3.8` and use `pypy3` instead of `pypy3.6`, which [apparently doesn't exist](https://travis-ci.org/algernon/adderall/jobs/648975808#L164).